src/itest/kotlin/nl/lifely/zac/itest/ReferentieTabelRestServiceTest.kt

```diff
@@ -43,8 +43,8 @@ class ReferentieTabelRestServiceTest : BehaviorSpec({
                 with(JSONArray(responseBody)) {
                     length() shouldBe 8
                     get(0) shouldBe "Balie"
```

Comment:
I thought this could be verified in a single collection `shouldContain` or somthing..

Reply:
you are right! did that now